src/main/java/com/royalrangers/bean/UserUpdate.java

```diff
@@ -0,0 +1,23 @@
+package com.royalrangers.bean;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.royalrangers.enums.UserAgeGroup;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@JsonSerialize
+public class UserUpdate {
```

Comment:
UserBean contains service fields like: enabled, confirmed, approved, which should not be changed by this api. So, in security reason api use this bean for accept only some fields.